eclipse-collections/src/main/java/org/eclipse/collections/impl/collector/SummaryStatistics.java

```diff
@@ -25,6 +25,8 @@
 
 /**
  * A Summarizer can be used to aggregate statistics for multiple primitive attributes.
+ *
+ * @since 8.1
  */
 public class SummaryStatistics<T> implements Procedure<T>
```

Comment:
I may have come up with a solution to this problem.  The SummaryStatistics classes in JDK are not final, so I can extend them and should be able to make them Serializable.  I'm working on it now.  If I get it to work I will commit an update later today.